README.md

```diff
@@ -1,17 +1,82 @@
+- **Flexible Data Schemas:** Storyblok's component-based approach gives you full control over your content structure. You can define your own custom data models with nestable components that can be reused across all your projects.
+- **Publish Faster:** Marketing and content teams work faster with a visual editor enabling them to work independently. No more content tickets in your backlog.
+- **Enterprise security**: ensure data security on one of the most secure, enterprise-grade CMS available on the market. Storyblok is ISO 27001 certified.
+- **Deliver Better Experiences**: Personalize content experiences for different markets, languages, or audience segments using Storyblok's built-in localization tools, or integrate with any third-party solution through API-first integrations.
+
+# How it works
+
+A **headless CMS** is a back-end-only content management system (CMS) built from the ground up as a content repository that makes content accessible via APIs.  [Headless CMS explained](https://www.storyblok.com/tp/headless-cms-explained)
+
+
+# Key Features
+
+**Composable Architecture:**
+
+- **APIs**: Work with any frontend or third-party integration with [GraphQL](https://www.storyblok.com/docs/graphql-api), [REST](https://www.storyblok.com/docs/api/content-delivery/v2), and [Management API.](https://www.storyblok.com/docs/api/management)
+- **Flexible Backend**: As a headless content management system, Storyblok can manage and serve content for any app or site.
+- **Apps**: Add extra functionalities to your space through a suite of custom apps available for free on the Storyblok App store, or build your own.
+- **Composable Components**: Define new content types such as an about page, hero section, gallery, and any other content type or content schema. Create once, reuse everywhere. The limit is your imagination.
+
+**Streamline Workflows:**
+
+- **Visual editing interface**: Storyblok enables non-technical users to build and edit JSON structures with an easy-to-use visual interface.
+- **Component Library**: Find, access, and edit components directly from the visual editor making it easier for teams to collaborate.
+- **Collaboration workflows**: Publish faster with inclusive collaboration workflows that give anyone on the team a simple way to make a suggestion or take action.
+
+**Audience Experiences:**
+
+- **Image Service**: Make apps and websites faster by Transforming, optimizing, and caching images in a CDN
+- **Internationalization**: Create personalized content experiences via built-in folder-level and field-level internationalization
+- **Omnichannel Experiences**: Storyblok’s headless architecture enables developers to publish content across any digital channel via APIs.
+
+# Resources
+
+- [Tutorials](https://www.storyblok.com/tutorials)
+- [FAQs](https://www.storyblok.com/faqs)
+- [Discord](https://discord.gg/jKrbAMz)
+- [Community](https://www.storyblok.com/community)
+- [Learning Hub](https://www.storyblok.com/docs)
```

Comment:
Can we also add the "Clients" as a section? js, ruby, python, php

Reply:
This will be [PHP SDK](link to php client)? @manuelschroederdev